src/vue/backend/src/components/Feedback.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div class="idx-wp-feedback">
```

Comment:
Since we can use the VCL, these should be idx-blocks so we can have the dynamic class prefixing